src/components/v5/common/Modals/InviteMembersModal/InviteMemberModal.tsx

```diff
@@ -54,43 +76,76 @@ const InviteMembersModal = ({ isOpen, onClose }: Props) => {
     `/invite/${colonyName}/${colonyMemberInvite?.id}`,
   );
   const invitesUsed = 100 - (colonyMemberInvite?.invitesRemaining || 0);
+  const isOutOfInvites = invitesUsed >= invitesAvailable;
```

Comment:
I think we could make use of the `const { totalMemberCount } = useMemberContext();` here to compute how many invites are used and `colonyMemberInvite?.invitesRemaining` for deciding how many invites are available, thus replacing lines `74-79` with
```
  const inviteLink = useBaseUrl(
    `/invite/${colonyName}/${colonyMemberInvite?.id}`,
  );
  const { totalMemberCount } = useMemberContext();
  const invitesAvailable = colonyMemberInvite?.invitesRemaining || 0;
  const invitesUsed = totalMemberCount;
  const isOutOfInvites = invitesUsed >= invitesAvailable;
```

Reply:
@arrenv do we want to hardcode the invites limit to `100`?
I think it would be best to actually use the stored remaining invites, as well as the number of existing followers and decide based on that if we are or are not out of invites

Reply:
@CzarekDryl no, I was not aware this was hardcoded, this is a value that can be increased in the database. As a result of this and to minimise impact on the database, we updated the design last week to remove the need to show the upper limit and just show the remaining available.